#!/usr/bin/env python
PACKAGE = "vision_module"

from dynamic_reconfigure.parameter_generator_catkin import *
 
gen = ParameterGenerator()
 
gen.add("NUM_PICT", int_t, 0, "NUM PICT TO SAVE", 15, 1, 30)
gen.add("FACE_LEARN", bool_t, 0, "FACE LEARN MODE", False)
gen.add("DISPLAY", bool_t, 0, "DISPLAY RESULT", False)
gen.add("SEND_IMAGE", bool_t, 0, "SAVE IMAGE", False)
gen.add("SAVE_IMAGE", bool_t, 0, "SAVE IMAGE", False)

exit(gen.generate(PACKAGE, "face_recognition", "face_recognition"))
